lib/http2/connection.c

```diff
@@ -401,7 +401,8 @@ static ssize_t expect_continuation_of_headers(h2o_http2_conn_t *conn, const uint
             conn->_read_expect = expect_default;
             if (stream->state == H2O_HTTP2_STREAM_STATE_RECV_HEADERS) {
                 hret = handle_incoming_request(conn, stream, (const uint8_t *)conn->_headers_unparsed->bytes,
-                                               conn->_headers_unparsed->size, err_desc);
+                                               conn->_headers_unparsed->size, (frame.flags & H2O_HTTP2_FRAME_FLAG_END_STREAM),
```

Comment:
Ah, my bad. I was adding CONTINUATION frames support to `h2get`, and I noticed that my local H2O was timing out, whereas www.google.com or www.facebook.com where responding as I expected. I pushed a small repro case here: